validator/webui/README.md

```diff
@@ -30,7 +30,7 @@ $ ./serve-standalone
 Then visit your own instance at http://127.0.0.1:8765/.
```

Comment:
This is being flagged as a dead link. Escaping it with backticks (`http://127.0.0.1:8765/`) will fix the problem.

Reply:
Thank you for the solution!